### PR TITLE
ign -> gz Provisional Finale: Source Migration : gz-plugin

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/767

### CODE REVIEW

File: loader/src/cmd/plugin_main.cc [hunk 1]
```diff
@@ -89,6 +89,6 @@ int main(int argc, char** argv)
 
   addPluginFlags(app);
 
-  app.formatter(std::make_shared<IgnitionFormatter>(&app));
+  app.formatter(std::make_shared<GzFormatter>(&app));
```
Comment (chapulina) [inline loader/src/cmd/plugin_main.cc:92]: Requires https://github.com/gazebosim/gz-utils/pull/70

**Review by ahcorde — DISMISSED** [review]
conflicts

File: MigrationFromCommon.md [hunk 2]
```diff
@@ -39,23 +39,23 @@ then you should link against the `register` component. This should almost always
...
-* `IGN_COMMON_REGISTER_SINGLE_MACRO` can be directly replaced with `GZ_ADD_PLUGIN`.
-* `IGN_COMMON_ADD_PLUGIN` can be directly replaced with `GZ_ADD_PLUGIN`.
-* All uses of `IGN_COMMON_BEGIN_ADDING_PLUGINS` can simply be removed.
-* All uses of `IGN_COMMON_FINISH_ADDING_PLUGINS` can simply be removed.
-* All uses of `IGN_COMMON_SPECIALIZE_INTERFACE` can simply be removed. Interfaces no longer need to be "specialized" in order to have specialized plugins.
+* `IGNITION_COMMON_REGISTER_SINGLE_MACRO` can be directly replaced with `GZ_ADD_PLUGIN`.
+* `IGNITION_COMMON_ADD_PLUGIN` can be directly replaced with `GZ_ADD_PLUGIN`.
+* All uses of `IGNITION_COMMON_BEGIN_ADDING_PLUGINS` can simply be removed.
+* All uses of `IGNITION_COMMON_FINISH_ADDING_PLUGINS` can simply be removed.
+* All uses of `IGNITION_COMMON_SPECIALIZE_INTERFACE` can simply be removed. Interfaces no longer need to be "specialized" in order to have specialized plugins.
```
[the remark this answers is not in the document]
Comment (methylDragon) [reply MigrationFromCommon.md:58]: Reverted!

**Comment by methylDragon** [comment]
> conflicts

Rebased and resolved! (I was wondering why the suffix became `pre1` though, but I guess it brings it in line with the others)

**Comment by methylDragon** [comment]
CI tests were coming back with linting errors.
Then I realized I could hard-tock the `DETAIL_IGNITION_PLUGIN_VISIBLE` macro to fix that https://github.com/gazebosim/gz-plugin/pull/93/commits/ba71a7a7cd811b63f654afcf69a0c198a761877d

**Review by chapulina — APPROVED** [review]

File: core/CMakeLists.txt [hunk 3]
```diff
@@ -17,4 +17,4 @@ gz_build_tests(
 
 
 add_subdirectory(include/gz/plugin)
-install(DIRECTORY include/ignition DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
```
Comment (scpeters) [inline core/CMakeLists.txt:20]: I think this broke installation of the `include/ignition` redirect headers

**Pull request merged**